data_model/1.1/clusters/OnOff.xml

```diff
@@ -133,12 +100,14 @@ Davis, CA 95616, USA
     </attribute>
     <attribute id="0x4001" name="OnTime" type="uint16" default="0">
       <access read="true" write="true" readPrivilege="view" writePrivilege="operate"/>
+      <quality nullable="true"/>
```

Comment:
huh...interesting. i wonder if this was ever actually true. I suppose in a way it doesn't totally matter, but changing from nullable to not nullable is not strictly backwards compatible